community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/main.tf

```diff
@@ -294,6 +294,14 @@ data "local_file" "setup_external" {
   filename = "${path.module}/files/setup_external.sh"
 }
 
+data "local_file" "main_task_prolog" {
```

Comment:
I'm not quite sure why this file needs to be individually uploaded to GSC bucket,
If we put it into `scripts/tools` folder it will be installed without extra steps.
Probably, `gpu-test` served as a bad example, we should change it as well (not in this PR).

Reply:
Thank you. Fixed.